scripts/modules.gd

```diff
@@ -40,36 +40,38 @@ func register_keyword(keyword: StringName) -> void:
 	Blueprint.all_keywords.append(keyword)
 
 
-## Registers a new rarity to be used in-game.
-func register_rarity(rarity: StringName, color: Color) -> void:
-	Blueprint.all_rarities.append(rarity)
-	Blueprint.all_rarity_colors[rarity] = color
-
-
 ## Requests the modules to respond to a request.
 func request(what: StringName, info: Array = []) -> bool:
-	print_verbose("[Modules] Requested %s with the following info: %s" % [what, info])
+	#print_verbose("[Modules] Requested %s with the following info: %s" % [what, info])
 	await Modules.wait_in_queue()
 	
 	requested.emit(what, info)
 	
-	print_verbose("[Modules] Waiting for response...")
+	#print_verbose("[Modules] Waiting for response...")
 	var modules_result: Dictionary = await Modules.wait_for_response()
-	print_verbose("[Modules] Modules Responded to %s. Parsing response..." % what)
+	#print_verbose("[Modules] Modules Responded to %s. Parsing response..." % what)
 	
 	if modules_result.is_empty():
 		# No CSCs in modules.
-		print_verbose("[Modules] No Modules Responded. Passing...")
+		#print_verbose("[Modules] No Modules Responded. Passing...")
 		return false
 	
 	var modules_response: bool = modules_result.result
 	var modules_amount: int = modules_result.amount
 	
-	print_verbose("[Modules] %d Modules Responded. Result: %s\n" % [modules_amount, modules_response])
+	#print_verbose("[Modules] %d Modules Responded. Result: %s\n" % [modules_amount, modules_response])
```

Comment:
Remove verbose printing since it was spamming the console with ~10,000 lines of garbage because of the "Update Card" request.